examples/stm32/f4/stm32f429i-discovery/lcd-serial-touch/README

```diff
@@ -0,0 +1,8 @@
+README lcd-serial-touch
+-----------------
+
+This example is based on lcd-serial. The serial part is not used anymore, but it remains in case someone is interested to use it.
```

Comment:
then you probably shouldn't call it "lcd-serial-touch" if it doesn't have any serial in it....

Reply:
As the guy who originally wrote 'lcd-serial' :-) the serial part is how the chip talks with the LCD. In this case serially through the SPI interface, as opposed to in parallel through the LTDC interface.